spec/services/complete_moab_handler_check_exist_spec.rb

```diff
@@ -429,6 +429,46 @@
         end
       end
 
+      context 'there are two moabs for the druid' do
```

Comment:
this is a bit redundant in light of the `spec/jobs/moab_to_catalog_job_spec.rb` addition.  i actually wrote this one first, then thought it'd be slightly better if it were just a bit more end-to-end, but for some reason i was also hesitant to remove this, since this is the spec file for the meat of the implementation.

happy to trim one or the other if people would prefer.  i think i prefer the other slightly, if forced to choose.